### DISCUSSION TITLE
Unable to invoke request: javax.net.ssl.SSLHandshakeException

### DISCUSSION THREAD

**Discussion by dkoci** [quarkusio/quarkus#28010] [Q&A]
Hi.
I have created a rest client to call some https api. Have quarkus.tls.trust-all=true in application.properties, but get an error:

Unable to invoke request: javax.net.ssl.SSLHandshakeException

Using quarkus 2.12-Final version.

Can someone help me with that? Thank you

**Comment by gsmet** [comment]
Which REST Client extension are you using?

**Reply by dkoci** [reply]
I am using `quarkus-rest-client` and `quarkus-rest-client-jackson` extensions.
